tests/testthat/00_create_test_data.R

```diff
@@ -20,13 +20,11 @@ test_that("test data can be created", {
   
   saveRDS(webprocess(DELIMITER = 'TAB'), "data/test_webprocess_tab.rds")
   
-  gconfig('wps.url'="https://cida.usgs.gov/gdp/process/WebProcessingService")
+  gconfig('wps.url'="https://labs.waterdata.usgs.gov/gdp-process-wps/WebProcessingService")
   wp <- webprocess()
   saveRDS(wp, "data/test_webprocess_knife_prod.rds")
   saveRDS(query(wp, 'algorithms'), "data/test_webprocess_algorithms_prod.rds")
   
-  # gconfig(wps.url = "https://cida-test.er.usgs.gov/gdp/process/WebProcessingService")
```

Comment:
Just removing these multi-tier tests, we don't intend to do active development on the GDP going forward so a test tier will be unlikely.